### ISSUE TITLE
Accuracy comparison

### ISSUE THREAD

**Issue by WJinhai** [kurapan/EAST#22]
Have you ever compared the accuracy between EAST using tensorflow and  this algorithm?

**Comment by kurapan** [comment]
I haven't done any thorough comparison with the objective to compare the performance, but I have used argman's TF implementation and was able to achieve F1 score of about 0.804 on IC15 in the best case. This Keras implementation achieves about 0.802, but I believe that there is some room for improvement if you spend some time on hyperparameter tuning of both the training phase and the final detection process (thresholding, minimum region area criterium, etc.).
